pkg/assert/pod.go

```diff
@@ -101,7 +102,7 @@ func ValidatePodIPByLabel(labels, expected []string) {
 func ValidatePodIPsByLabel(label string, expected []string) {
 	cmd := "kubectl get pods -l " + label +
 		` -o jsonpath='{range .items[*]}{.status.podIPs[*].ip}{" "}{end}'` +
-		" --kubeconfig=" + shared.KubeConfigFile
+		" --kubeconfig=" + factory.KubeConfigFile
```

Comment:
can we store and fetch KubeConfigFile from cluster var?

Reply:
this one as per how its used now, i would say lets keep global